server/api/src/delayed_init.py

```diff
@@ -4,10 +4,11 @@
 import os
```

Comment:
I have a feeling that now, when we exactly know what works and what doesn't work, would be a great moment to rewrite this whole file in a more readable way : )

I would think about some `Yagna` class (as mentioned in the first comment in `app.py`) that does all that stuff on initialization.